# HG changeset patch
# Parent a888f210af4e222fdb083a96b76c82f75b076e58
# User Frédéric Wang <fred.wang@free.fr>
Remove #pragma GCC optimize ("O0") (compilation failure because of a gcc bug)

diff --git a/netwerk/protocol/http/HttpChannelParent.cpp b/netwerk/protocol/http/HttpChannelParent.cpp
--- a/netwerk/protocol/http/HttpChannelParent.cpp
+++ b/netwerk/protocol/http/HttpChannelParent.cpp
@@ -331,17 +331,16 @@ HttpChannelParent::RecvUpdateAssociatedC
   return true;
 }
 
 // Bug 621446 investigation, we don't want conditional PR_Aborts bellow to be
 // merged to a single address.
 #ifdef _MSC_VER
 #pragma warning(disable : 4068)
 #endif
-#pragma GCC optimize ("O0")
 
 bool
 HttpChannelParent::RecvRedirect2Verify(const nsresult& result, 
                                        const RequestHeaderTuples& changedHeaders)
 {
   if (NS_SUCCEEDED(result)) {
     nsCOMPtr<nsIHttpChannel> newHttpChannel =
         do_QueryInterface(mRedirectChannel);
